app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -53,7 +53,7 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         loginPresenter.onAttach(this)
     }
 
-    override fun onLoginSuccess(message: String) {
+    override fun onLoginSuccess(message: String?) {
```

Comment:
It was working earlier without a "!!" at the end. Any change which generates a need to add a "?" or a "!!"